sdk/core/core-util/test/public/aborterUtils.spec.ts

```diff
@@ -135,18 +135,12 @@ describe("cancelablePromiseRace", function () {
   });
 
   it("should respect the abort signal supplied", async function () {
-    const aborter = new AbortController();
-    setTimeout(() => aborter.abort(), function1Delay / 2);
-    let errorThrown = false;
-    try {
-      await cancelablePromiseRace<[number, string, void]>([function1, function2, function3], {
-        abortSignal: aborter.signal,
-      }); // all are aborted
-    } catch (error) {
-      errorThrown = true;
-      assert.strictEqual((error as { message: string }).message, "The operation was aborted.");
-    }
-    assert.isTrue(errorThrown);
+    const signal = AbortSignal.timeout(function1Delay / 2);
+    await expect(
+      cancelablePromiseRace<[number, string, void]>([function1, function2, function3], {
+        abortSignal: signal,
+      }),
+    ).rejects.toThrowError(AbortError);
```

Comment:
I'd like to change this everywhere - before I do so any concerns?

We can also have `rejects.toThrowError(new AbortError("This operation was aborted"))` if we want to assert on the message contents as well. It just makes for more brittle tests